builder/bundles.go

```diff
@@ -240,6 +243,8 @@ func resolveFiles(numWorkers int, set bundleSet, bundleRepoPkgs map[string]map[s
 				errorCh <- err
 				break
 			}
+			pkgMutex.Unlock()
```

Comment:
move this to before the error checking.

Reply:
Actually, this doesn't need a mutex, it just needs a different name for the `err` variable so all goroutines aren't using the same one.